src/main/java/com/scottlogic/strategy/FilterStrategy.java

```diff
@@ -0,0 +1,9 @@
+package com.scottlogic.strategy;
+
+import com.scottlogic.UserPost;
+
+import java.util.List;
+
+public interface FilterStrategy {
```

Comment:
You can just call this interface Filter.
You can apply strategy without naming it explicitly (avoids tying your implementation to some pattern, which may become outdated or too restrictive)

Reply:
Shall I apply the same to the 'SortStrategy' interface?

Reply:
Yes sure, would be more consistent.
I allowed it before so you could have some time to realise that it was the strategy class, but in practice it is not usually named.

Reply:
Sounds good, fixed